cds/modules/deposit/static/js/cds_deposit/avc/components/cdsForm.js

```diff
@@ -267,6 +232,22 @@ function cdsFormCtrl($scope, $http, $q, schemaFormDecorators) {
   this.autocompleteType = function() {
     return that.types.promise;
   }
+
+  /**
+   * Utilities
+   */
+  this.stripCommas = function(string) {
```

Comment:
Does the same apply for `autocomplete`, `formKeyword`, `formAuthor`, `authorFromUser`, etc...?
Also, should I just declare them as var in the same position?